--- Running scipy.fft.dct#1YY/dctYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.fft.dct#1YY'), ('notebook', 'dctYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.460s
user	0m1.814s
sys	0m6.919s
